<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreTab</class>
 <widget class="QWidget" name="StoreTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="id">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="name">
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="type">
       <property name="statusTip">
        <string extracomment="The type of store. Currently only default is available."/>
       </property>
       <item>
        <property name="text">
         <string>default</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Save Inventory Every</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="saveEvery">
       <property name="statusTip">
        <string extracomment="The interval in minutes the server should save this stores inventory levels. Set to 0 to disable."/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Restock Inventory Every</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="restockEvery">
       <property name="statusTip">
        <string extracomment="The interval in minutes the server should restock inventory levels. Set to 0 to disable."/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Allowed Conditions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QListWidget" name="allowedConditions">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="statusTip">
        <string extracomment="The allowed conditions the store will accept from players."/>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <item>
        <property name="text">
         <string>Prestine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Worn</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Damaged</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Badly Damaged</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="logTransactions">
       <property name="statusTip">
        <string extracomment="Check to enable server level logging of transactions."/>
       </property>
       <property name="text">
        <string>Log Transactions</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="showOutOfStock">
       <property name="statusTip">
        <string extracomment="Check to allow the store to display out of stock items to players."/>
       </property>
       <property name="text">
        <string>Show Out of Stock</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
